core/Azure.Mcp.Core/src/Areas/Server/Resources/consolidated-tools.json

```diff
@@ -2196,6 +2196,189 @@
       "mappedToolList": [
         "speech_tts_synthesize"
       ]
+    },
+    {
+      "name": "manage_azure_storage_sync_services",
```

Comment:
We should split the entries here by intent, not just by resource. For each File Sync resource type, create separate tool groups for distinct action types:

Get/List only:
get_azure_storage_sync_services
get_azure_storage_sync_groups
get_azure_storage_sync_cloud_endpoints
get_azure_storage_sync_registered_servers
get_azure_storage_sync_server_endpoints


Create:
create_azure_storage_sync_service

(similarly for other types)

Edit/Update:
edit_azure_storage_sync_service
etc.

Delete:
delete_azure_storage_sync_service
etc.


Look at the other intent-based entries in the file for reference.